### PR TITLE
nitclk documentation cleanup

### LINKED ISSUE #1225: `nitclk` documentation problems
* says "list of list of"... Should be "list of" and the int needs to go away.
* TBDs need to be filled out
* SessionReference constructor add snippet "Clients should have no need to construct objects of this type directly" or something like that.
* Properties of type SessionReference need Datatype documentation updated

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] ~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~
- [X] ~~I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?
* Clean up nitclk documentation in preparation for 1.0
    * Update parameter type for documentation
        * Remove reference to the session number since it is no longer supported
        * Fix how it is displayed since it is a list
    * Update SessionReference type properties type in documentation to remove session number
    * Replace TBD documentation with string from C help

### List issues fixed by this Pull Request below, if any.
* Fix #1225 

### What testing has been done?
* Visual inspection of generated files

### CODE REVIEW

File: src/nitclk/templates/class.rst.mako [hunk 1]
```diff
@@ -63,10 +63,7 @@ ${helper.get_rst_header_snippet('SessionReference', '=')}
     Helper class that contains all NI-TClk properties. This class is what is returned by
     any nimi-python Session class tclk attribute when the driver supports NI-TClk
 
-    .. code:: python
-
-        with niscope.Session('dev1') as session:
-            session.tclk.sample_clock_delay = .42
```
Comment (sbethur) [inline src/nitclk/templates/class.rst.mako]: Why remove the code snippet?

File: generated/nitclk/nitclk/session.py [hunk 2]
```diff
@@ -73,13 +76,13 @@ class SessionReference(object):
     Default Value is empty string
     '''
     pause_trigger_master_session = _attributes.AttributeSessionReference(6)
-    '''Type: nimi-python Session class, nitclk.SessionReference, NI-TClk Session Number
+    '''Type: nimi-python Session class or nitclk.SessionReference
```
Comment (marcoskirsch) [inline generated/nitclk/nitclk/session.py]: * I wonder if "nimi-python Session class" makes sense to users.
* Why have "class" there but not after `nitclk.SessionReference`?

Not sure best way to reword. Maybe just "Driver Session or nitclk.SessionReference" ?

Comment (texasaggie97) [reply generated/nitclk/nitclk/session.py]: I don't have a better way either so I will just use "Driver Session or nitclk.SessionReference"

File: src/nitclk/templates/class.rst.mako (hunk 1 above)
Comment (texasaggie97) [reply src/nitclk/templates/class.rst.mako]: Added back

**Pull request merged**